README.md

```diff
@@ -478,48 +479,145 @@ Consider `model_8` which is nested in the `marts` subdirectory:
 
 This model should be renamed to either `fct_model_8` or `dim_model_8`.
 
------
-### Model Test Directories
+### Model Directories
 #### Model
 
-`fct_tests_directories` ([source](models/marts/structure/fct_tests_directories.sql)) shows all cases where model tests are NOT in the same subdirectory as the corresponding model.
+`fct_model_directories` ([source](models/marts/structure/fct_model_directories.sql)) shows all cases where a model is NOT in the appropriate subdirectory:
+- For staging models: The files should be in nested in the staging folder in a subfolder that matches their source parent's name.
+- For non-staging models: The files should be nested closest to the folder name that matches their model type.  
 
 #### Reason to Flag
 
-Each subdirectory in `models/` should contain one .yml file that includes the tests and documentation for all models within the given subdirectory. Keeping your repository organized in this way ensures that folks can quickly access the information they need.
+Because we often work with multiple data sources, in our staging directory, we create one directory per source.
```

Comment:
suggestion: should this be subdirectory?
```suggestion
Because we often work with multiple data sources, in our staging directory, we create one subdirectory per source.
```